shared/chat/conversation/input-area/normal/platform-input.desktop.tsx

```diff
@@ -132,13 +128,10 @@ class _PlatformInput extends React.Component<PlatformInputPropsInternal, State>
   _insertEmoji = (emojiColons: string) => {
     if (this._input) {
       this._input.transformText(({text, selection}) => {
-        const newText = text.slice(0, selection.start) + emojiColons + text.slice(selection.end)
-        const pos = selection.start + emojiColons.length
+        const newText = text.slice(0, selection.start || 0) + emojiColons + text.slice(selection.end || 0)
```

Comment:
selection can actually be null. we could enforce this check way higher but i did this for now

Reply:
if selection can be null, wouldn't something like `selection.start` cause a property undefined error?

Reply:
selection isn't null. selection.start can be